Assets/GothicVR/Scripts/Creator/Meshes/AbstractMeshCreator.cs

```diff
@@ -510,5 +521,29 @@ protected Material GetEmptyMaterial()
 
             return material;
         }
+
+        private Material GetWaterMaterial(PxMaterialData materialData)
```

Comment:
Well done. :-)

Reply:
That was your implementation =D
I simply modified it for the abstractmeshcreator.